indico_jacow/plugin.py

```diff
@@ -179,6 +188,10 @@ def _add_sidemenu_item(self, sender, event, **kwargs):
             return
         return SideMenuItem('abstracts_stats', _('CfA Statistics'),
                             url_for_plugin('jacow.abstracts_stats', event), section='reports')
+    
+    def _extend_user_profile_menu(self, sender, user, **kwargs):
+        yield SideMenuItem('mailing_lists', _('Mailing Lists'),
```

Comment:
Doesn't really make a difference but if it's just one item you can also `return` it